pallets/runtime/tests/src/settlement_test.rs

```
@@ -86,7 +84,7 @@ fn create_token(token_name: &[u8], ticker: Ticker, keyring: Public) {
 fn next_block() {
     let block_number = System::block_number() + 1;
     System::set_block_number(block_number);
-    Settlement::on_initialize(block_number);
+    let _ = Scheduler::on_initialize(block_number);
```

since this is a test, we should panic on err instead of ignoring the result.

---

The returned value is of type `Weight`, not a `Result` :smile:

---

We can simply ignore it then.

---

`let _ = ...` simply avoids a Clippy warning.